content/en/docs/architecture/step-registry.md

```diff
@@ -959,3 +959,19 @@ tests:
   steps:
     workflow: openshift-e2e-disruptive
 {{< / highlight >}}
+
+## VPN connection
+
+For platforms that need access to restricted environments, `ci-operator`
+supports adding a dedicated VPN connection to each test step.  Since this is a
+requirement of specific platforms, it is enabled when a
+[cluster profile]({{< ref "docs/how-tos/adding-a-cluster-profile" >}})
+for one of those platforms is used.  This process is transparent to the test
```

Comment:
That would require some rearchitecting since this is all based on a cluster profile and I don't see a simple way of adapting that to claims.

But I think that is more of a statement about the need to disentangle the mess that are profiles than a problem with claims or VPNs.